docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -76,7 +76,7 @@ Every host will always belong to at least 2 groups (``all`` and ``ungrouped`` or
 Hosts in multiple groups
 ------------------------
 
-You can (and probably will) put each host in more than one group. For example a production webserver in a datacenter in Atlanta might be included in groups called ``[prod]`` and ``[atlanta]`` and ``[webservers]``. You can create groups that track:
+You can (and probably will) put each host in more than one group. For example, a production webserver in a datacenter in Atlanta might be included in groups called ``[prod]`` and ``[atlanta]`` and ``[webservers]``. You can create groups that track:
```

Comment:
```suggestion
You can put each host in more than one group. For example, a production web server in a data center in Atlanta might be included in groups called ``[prod]`` and ``[atlanta]`` and ``[webservers]``. You can create groups that track:
```

Reply:
I guess it's kind of scope creep but I'd suggest we make these changes while we're in here.